lib/v2/curius/maintainer.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/link/:name?': ['Ovler-Young'],
```

Comment:
The key here should be the same as `router.js` as well as `path` in the docs

Reply:
Fixed